src/theme/components/index.tsx

```diff
@@ -99,9 +99,11 @@ export const ClickableStyle = css`
   }
 `
 
-export const LinkStyle = css`
-  color: ${({ theme }) => theme.accentAction};
-  stroke: ${({ theme }) => theme.accentAction};
+export const LinkStyle = css<{ $uniformColorInLightAndDarkMode?: boolean }>`
+  color: ${({ theme, $uniformColorInLightAndDarkMode }) =>
+    $uniformColorInLightAndDarkMode ? theme.accentActive : theme.accentAction};
+  stroke: ${({ theme, $uniformColorInLightAndDarkMode }) =>
+    $uniformColorInLightAndDarkMode ? theme.accentActive : theme.accentAction};
```

Comment:
Not sure about this one, to be honest! I am quite confused if this isn't just an NFT specific exception. Anyway, for my PR, I did this: https://github.com/Uniswap/interface/pull/5417

Reply:
I think going forward we need to develop much better abstraction for this, which is similar to what we have to do for Text components as well! 

Meanwhile, I think this is better while handled on a component-level, so that we don't introduce a new API that can dangerously lead to our links design getting inconsistent!

Reply:
done! thanks!